SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -148,6 +148,9 @@
         EMPhysics   = cms.untracked.bool(True),
         HadPhysics  = cms.untracked.bool(True),
         FlagBERT    = cms.untracked.bool(False),
+        LowEnergyGflashEcal = cms.bool(False),
+        LowEnergyGflashEcalEmin = cms.double(0),
```

Comment:
@dmitrySorokin , can Emin be non zero?

Reply:
Yes. One can apply this parametrization in any range. 
Like in GFlash it works from 50 GeV to 10 GeV.